README.md

```diff
@@ -6,3 +6,4 @@
 | [Console Replay Guide](https://github.com/project-slippi/slippi-wiki/blob/master/CONSOLE_REPLAYS.md) | Guide to setting up a Wii to generate replays. |
 | [Console Mirroring Guide](https://github.com/project-slippi/slippi-wiki/blob/master/CONSOLE_MIRRORING.md) | Guide to setting up a Wii to mirror gameplay on a computer. |
 | [Direct Connect Guide](https://github.com/project-slippi/slippi-wiki/blob/master/DIRECT_CONNECT.md) | Guide to using a computer in place of a router for the wii network connection. |
+|[Spectator Protocol](SPECTATOR_PROTOCOL.md)| Description of the networking protocol that Slippi's Dolphin uses for spectating.|
```

Comment:
probably mention that console uses the same overall protocol

Reply:
Added